resident-default.properties

```diff
@@ -1142,4 +1145,17 @@ resident-transliteration-workaround-for-eng-fra=eng-hin,hin-fra
 resident-transliteration-workaround-for-eng-spa=eng-hin,hin-spa
 
 # Reg-processer-credential-partner-policy-url
-mosip.resident.reg-processer-credential-partner-policy-url=${config.server.file.storage.uri}registration-processor-credential-partners.json
\ No newline at end of file
+mosip.resident.reg-processer-credential-partner-policy-url=${config.server.file.storage.uri}registration-processor-credential-partners.json
+
+hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
+## Keymanager service
+#Type of keystore, Supported Types: PKCS11, PKCS12, Offline, JCE
+mosip.kernel.keymanager.hsm.keystore-type=OFFLINE
+# For PKCS11 provide Path of config file.
+# For PKCS12 keystore type provide the p12/pfx file path. P12 file will be created internally so provide only file path & file name.
+# For Offline & JCE property can be left blank, specified value will be ignored.
+mosip.kernel.keymanager.hsm.config-path=/config/softhsm-application.conf
+# Passkey of keystore for PKCS11, PKCS12
+# For Offline & JCE proer can be left blank. JCE password use other JCE specific properties.
+mosip.kernel.keymanager.hsm.keystore-pass=${softhsm.kernel.security.pin}
+mosip.identity.get.excluded.attribute.list=UIN,verifiedAttributes,IDSchemaVersion
```

Comment:
Is this property relevant to resident ? Looks more of idrepo property.

Reply:
https://mosip.atlassian.net/browse/MOSIP-34475 Added this check as part of this bug.